core/ipc/rpc/utils/src/main/java/org/opennms/core/rpc/utils/mate/EntityScopeProviderImpl.java

```diff
@@ -51,12 +55,17 @@
 
 public class EntityScopeProviderImpl implements EntityScopeProvider {
 
+    private static final String INTERFACE = "interface";
```

Comment:
Makes sense. Can you do that for the other context names, too. And maybe move them to the interface?

Reply:
:+1: done